### ISSUE TITLE
Runtime error while running example/application_object_detection.py

### ISSUE THREAD

**Issue by shuoli90** [Trusted-AI/adversarial-robustness-toolbox#1347]
**Describe the bug**
While running application_object_detection.py, I encountered an error. I have attached the error message 

**To Reproduce**
- Download corresponding images from the internet;
- Run "python example/application_object_detection.py"

**Expected behavior**
An error message will be given.

**Screenshots**
![Screen Shot 2021-10-06 at 9 03 21 PM](https://user-images.githubusercontent.com/19268975/136304220-e47822ce-84bf-448b-a4cc-3a2666bbff5c.png)

**System information (please complete the following information):**
 - OS:
 - Python version: 3.8.0
 - ART: 1.8.0
 - Torch: 1.9.1+cu102
 - Torchvisio: 0.10.1+cu102

**Comment by beat-buesser** [comment]
Hi @shuoli90 Thank you very much for reporting this issue. We are aware of it and it is caused by `torchvision` not supporting gradients at the inputs in versions 1.8 and 1.9 of the pretrained object detection models. `PyTorchObjectDetector` contains a warning for 1.8, but not yet 1.9 which we'll update now. Btw, we have successfully contributed a solution to `torchvision`, which should be part of the next release of `torchvision` [#4327](https://github.com/pytorch/vision/pull/4327).

**Comment by shuoli90** [comment]
Just to follow up, currently, is there any way that I can use PyTorchObjectDetector?
